corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -311,6 +311,10 @@ def get_forms_by_type(domain, type_, limit, recent_first=False):
         return forms[:limit]
 
     @staticmethod
+    def get_number_of_forms_in_domain(domain, doc_type='XFormInstance'):
+        return len(FormAccessorSQL.get_form_ids_in_domain_by_type(domain, doc_type))
```

Comment:
These don't use `count`, but that's not really necessary as there's not any huge sql domains at the moment